packages/client/src/components/authForm/authForm.module.scss

```
@@ -0,0 +1,63 @@
+@import '../../styles/globals.scss';
+@import '../../styles/mixin.scss';
+
+.container {
+  width: 360px;
+  padding: 40px;
+  background-color: white;
+  border-radius: 10px;
+  @include shadowDefault;
+  text-align: center;
+  box-sizing: border-box;
+  margin: 10px 0;
+}
+
+.title {
+  @include h2;
+  color: $text-base;
+  margin-bottom: 36px;
+}
+
+.error {
+  @include text-base;
+  color: $text-error;
+  text-align: left;
+  margin: 36px 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input {
+  width: 100%;
+
+  & + .input {
+    margin-top: 32px;
+  }
+}
+
+.linkTitle {
+  @include text-base;
+  color: $text-additional;
+  margin: 0;
+  margin-top: 23px;
```

[Предложение]
Можно в одно свойство записать
```suggestion
  margin: 23px 0 0;
```